frontend/src/core/loaders/components/SkeletonLoader.css

```diff
@@ -0,0 +1,37 @@
+.skeleton-loader.entities {
+    pointer-events: none;
+    overflow: hidden;
+    height: 100%;
+}
+
+.skeleton-loader.loading {
+    pointer-events: none;
+    overflow: hidden;
```

Comment:
You don't need to repeat these two properties here. You should change the selector to `.skeleton-loader.entities.loading`.